crates/wallet/src/wallet/mod.rs

```diff
@@ -1502,10 +1502,12 @@ impl Wallet {
 
         let recipients = params.recipients.iter().map(|(r, v)| (r, *v));
 
+        #[allow(deprecated)]
         for (index, (script_pubkey, value)) in recipients.enumerate() {
             if !params.allow_dust
                 && value.is_dust(script_pubkey)
                 && !script_pubkey.is_provably_unspendable()
+            // FIXME: (@leonardo) I think it needs further discussion, should we rely on the suggested `is_op_return` ?
```

Comment:
Yes I'm in favor of using `is_op_return`

Reply:
Agreed, we should use `is_op_return()`, especially since `is_provably_unspendable()` is now deprecated.

Reply:
Great, I'll fix it to use it then.